pkg/tsdb/cloudwatch/metric_data_query_builder.go

```diff
@@ -9,10 +10,13 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/cloudwatch"
 
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/features"
 	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/models"
 )
 
-func (e *cloudWatchExecutor) buildMetricDataQuery(query *models.CloudWatchQuery) (*cloudwatch.MetricDataQuery, error) {
+const keySeparator = "|&|"
```

Comment:
Do we think this is a distinct enough separator? I didn't add spaces because it makes the parsing simpler for when there are no results and ${LABEL} is "" (AWS removes the leading space)